### ISSUE TITLE
Tommy Muldoon - Unique Starting Item

### ISSUE THREAD

**Issue by scrubbless** [NPBruce/valkyrie#673]
When selecting Tommy Muldoon as a character, the starting items does not show his unique item "Becky" like it does for Duke (Ashcan) and the Flux Stabalizer (Kate Winthrop?).

**Comment by scrubbless** [comment]
That character is also over powered, but you can't fix that ;)

**Comment by Hobbeslionheart** [comment]
In addition to that, I had a scenario where we started with Becky and I didn't select Tommy Muldoon. It's being lumped as a ordinary weapon for random selection. 

While a player could add it manually, I would remove it from inclusion in the traits somehow, I'm not sure I would want to have to remove Becky from the game each scenario I design if I want to stay true to the game mechanics.

**Comment by NPBruce** [comment]
Requiring items be common should prevent it from appearing.  Which scenario was this?

**Comment by Hobbeslionheart** [comment]
It was in my scenario, Down the Attic Hole. I looked at the pool traits and it said firearm. I looked through the list and it appears that there are no other weapons in the unique item pool for anything. So adding common as a pool trait or excluding Becky wouldn't be the worst thing in a world, but seems like a silly workaround to exclude one thing that would never happen in official scenarios (I suspect), 

That said, who knows what future expansions might bring and so doing this workaround could nip other weapons in the bud.
